migrations/2023-02-07-070512_change_merchant_connector_id_data_type/down.sql

```diff
@@ -0,0 +1,10 @@
+CREATE SEQUENCE IF NOT EXISTS merchant_connector_id_seq OWNED BY merchant_connector_account.merchant_connector_id;
+
+UPDATE merchant_connector_account
+SET merchant_connector_id = id;
```

Comment:
So here for a safe conversion from varchar to int we set connector_id to id and then convert to Integer